package.json

```diff
@@ -25,14 +25,17 @@
     "nuxt-buefy": "^0.3.2",
     "pm2": "^4.2.0",
     "stylus": "^0.54.7",
-    "stylus-loader": "^3.0.2"
+    "stylus-loader": "^3.0.2",
+    "vue": "^2.6.11"
```

Comment:
2.6.11 doesn't work for me

```
Vue packages version mismatch:

- vue@2.6.11
- vue-server-renderer@2.6.10

This may cause things to work incorrectly. Make sure to use the same version for both.
```

Reply:
I have changed the vue version to 2.6.10，and it could also worked in this version.